### DISCUSSION TITLE
Is there a plan to support multi-agent workflow

### DISCUSSION THREAD

**Discussion by aodashuai001** [langgenius/dify#7206] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Multi agent can cope with more scenarios. Have you considered supporting multi-agent

### 3. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by create-my** [comment]
YES，I NEED THIS FEATURE TOO。workflow only can add tools or LLM. Can't support add agent.
